modules/ROOT/partials/configuration/textpattern_patterns.adoc

```diff
@@ -8,8 +9,8 @@ ____
 IMPORTANT: Any formats or commands used by `textpatterns` need to be registered with the editor when it is initialized. This may include enabling relevant plugins, such as the `lists` plugin.
```

Comment:
This admonition needs converting into a block admonition. For example:
```
[IMPORTANT]
====
Any formats or commands used by `textpatterns` need to be registered with the editor when it is initialized. This may include enabling relevant plugins, such as the `lists` plugin.
For information on:

* Registering formats for {productname}, see: xref:configure/content-formatting.adoc#formats[Content formatting options - `formats`].
* Registering commands for {productname}, see: xref:apis/tinymce.editor.adoc#addCommand[{productname} APIs - EditorCommands].
====
```

Reply:
Done